docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-kestra.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Deploy with Kestra
+description: How to deploy a pipeline with Kestra
+keywords: [how to, deploy a pipeline, Kestra]
+---
+
+# Deploy with Kestra
+
+## Introduction to Kestra
+
+[Kestra](https://kestra.io/docs) is an open-source, scalable orchestration platform that enables
+all engineers to manage business-critical workflows declaratively in code. By applying 
```

Comment:
enables ~~all~~ engineers to manage . . .

Reply:
corrected